docs/sources/setup-grafana/configure-docker.md

```diff
@@ -8,48 +8,81 @@ keywords:
 
-### Run Grafana container using bind mounts
+The Ubuntu-based Grafana Enterprise and OSS images are built using the [Ubuntu](https://ubuntu.com/) base image, which can be found in the [Ubuntu docker repo](https://hub.docker.com/_/ubuntu). An Ubuntu-based image can be a good option for users who prefer an Ubuntu-based image or require certain tools unavailable on Alpine.
 
-You may want to run Grafana in Docker but use folders on your host for the database or configuration. When doing so, it becomes important to start the container with a user that is able to access and write to the folder you map into the container.
+- **Grafana Enterprise**: `grafana/grafana-enterprise:<version>-ubuntu`
+
+- **Grafana Open Source**: `grafana/grafana-oss:<version>-ubuntu`
+
+## Run a specific version of Grafana
+
+You can also run a specific version of Grafana or a beta version based on the main branch of the [grafana/grafana GitHub repository](https://github.com/grafana/grafana).
+
+> **Note:** If you use a Linux operating system such as Debian or Ubuntu and encounter permission errors when running Docker commands, you might need to prefix the command with `sudo` or add your user to the `docker` group. The official Docker documentation provides instructions on how to [run Docker without a non-root user](https://docs.docker.com/engine/install/linux-postinstall/).
+
+To run a specific version of Grafana, add it in the command <version number> section:
 
 ```bash
-mkdir data # creates a folder for your data
-ID=$(id -u) # saves your user id in the ID variable
+docker run -d -p 3000:3000 --name grafana grafana/grafana-enterprise:<version number>
+```
+
+Example:
+
+The following command runs the Grafana Enterprise container and specifies version 9.4.7. If you want to run a different version, modify the version number section.
 
-# starts grafana with your user id and using the data folder
-docker run -d --user $ID --volume "$PWD/data:/var/lib/grafana" -p 3000:3000 grafana/grafana-enterprise:8.2.1
+```bash
+docker run -d -p 3000:3000 --name grafana grafana/grafana-enterprise:9.4.7
 ```
 
+## Run the Grafana main branch
+
+After every successful build of the main branch, two tags, `grafana/grafana-oss:main` and `grafana/grafana-oss:main-ubuntu`, are updated. Additionally, two new tags are created: `grafana/grafana-oss-dev:-pre` and `grafana/grafana-oss-dev:-pre-ubuntu`, where version is the next version of Grafana and build ID is the ID of the corresponding CI build. These tags provide access to the most recent Grafana main builds.
+
+To ensure stability and consistency, we strongly recommend using the `grafana/grafana-oss-dev:-pre` tag when running the Grafana main branch in a production environment. This tag ensures that you are using a specific version of Grafana instead of the most recent commit, which could potentially introduce bugs or issues. It also avoids polluting the tag namespace for the main Grafana images with thousands of pre-release tags.
```

Comment:
You've lost the placeholders for version and build id here so the tags don't make sense. The correct format should be something like `grafana/grafana-oss-dev:<version><build ID>-pre`

You can see the tags here https://hub.docker.com/r/grafana/grafana-oss-dev/tags (and we should add that url as a link in this doc so users can too).

Reply:
Fixed the path to the dev images and also added the reference link